ompi/mca/osc/rdma/osc_rdma_component.c

```diff
@@ -175,10 +175,11 @@ static int ompi_osc_rdma_pvar_read (const struct mca_base_pvar_t *pvar, void *va
     ompi_win_t *win = (ompi_win_t *) obj;
     ompi_osc_rdma_module_t *module = GET_MODULE(win);
     int offset = (int) (intptr_t) pvar->ctx;
+    int ret;
 
-    memcpy (value, (char *) module + offset, sizeof (unsigned long));
+    ret = osc_rdma_accelerator_mem_copy(value, (char *) module + offset, sizeof (unsigned long));
```

Comment:
This will always copy from host memory.

Reply:
Addressed all the comments for osc_rdma_component.c and osc_rdma_peer.c